.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+on: 
+    pull_request:
+      branches: [ master ]
+    push:
+      branches: [ master ]
+name: Test
```

Comment:
This forms part of the action name, would this work?
```suggestion
name: ci/github
```

Reply:
updated in current push